server.go

```diff
@@ -3345,9 +3346,14 @@ func (s *Server) DiagnosticsAsRows() []*influxql.Row {
 
 		// Shard may not be local to this node.
 		if sh.store != nil {
-			shardsRow.Columns = append(shardsRow.Columns, "path")
-			shardsRow.Values[0] = append(shardsRow.Values[0], sh.store.Path())
+			if !hasPath {
```

Comment:
@otoolep but you iterate through each shard. The hasPath variable controls if if the path column has been added before or we need to add it as it's the first time it appears